app/middlewares/generics.js

```diff
@@ -0,0 +1,8 @@
+const logger = require('../logger');
+
+exports.logRequestInformation = (req, res, next) => {
```

Comment:
this is how you should defined middlewares, which is common functionality that most endpoints can use. There is really no need to do this though.

Reply:
Can I maintain it? To get more information about requests.